.travis.yml

```diff
@@ -59,7 +59,7 @@ jobs:
 
         - &STANDARD_TEST_JOB
             stage: Fast Test
-            php: 7.0
+            php: 7.4
```

Comment:
I hope this is okey. I use PHP 7.1 code in my test resources. 

https://travis-ci.org/github/FriendsOfPHP/PHP-CS-Fixer/jobs/671019065#L825